index.js

```diff
@@ -443,7 +443,7 @@ const fileType = input => {
 			check([0xFF, 0xF0], {offset: start, mask: [0xFF, 0xFC]}) // MPEG 4 layer 0 using ADTS
```

Comment:
`AAAAAAAA AAAABCCD`

Letter | Length (bits) | Description
-- | -- | --
A | 12 | syncword 0xFFF, all bits must be 1
B | 1 | MPEG Version: 0 for MPEG-4, 1 for MPEG-2
C | 2 | Layer: always 0
D | 1 | protection absent, Warning, set to 1 if there is no CRC and 0 if there is CRC

2nd byte check should be `0b11110000`(`0xF0`) with mask `0b11111110`(`0xFE`) for `aac` then(and remaining options caught separately).

Reply:
also, shouldn't mpeg2 be caught too?

Reply:
There actually already is an ADTS/MPEG-2/ACC [test file](https://github.com/sindresorhus/file-type/blob/master/fixture/fixture-faac-adts.mp2), originating from PR #136. Currently with `.mp2` extension, it looks like to reverse engineered the extension based on the MPEG version.

Okay, lets assume this is ext. `.acc`  mime `audio/acc` candidate as well.